resources/core/charts/service-catalog/charts/binding-usage-controller/templates/deployment.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       serviceAccountName: {{ template "fullname" . }}
       containers:
         - name: {{ .Chart.Name }}
-          image: "{{ .Values.global.containerRegistry.path }}/binding-usage-controller:{{ .Values.image.tag }}"
+          image: "{{ .Values.global.containerRegistry.path }}/snapshot/binding-usage-controller:{{ .Values.image.tag }}"
```

Comment:
this and value for **tag** property in `values.yaml` will be updated after merging and releasing BUC from this [PR](https://github.com/kyma-project/kyma/pull/246)
I've changed it to snapshot to be sure that E2E scenario is working